docs/uzl_omics.md

```diff
@@ -0,0 +1,32 @@
+# nf-core/configs: UzL OMICS Cluster Configuration
+
+The rnaseq nf-core pipeline has been successfully configured for use on the UzL OMICS cluster at the University of Luebeck.
+Implementation of nf-core pipelines for use on the UzL OMICS is in process. 
+
+To use, run the pipeline with `-profile uzl_omics`. This will download and launch the `uzl_omics.config` which has been pre-configured with a setup suitable for the UzL OMICS cluster. Using this profile, a docker image containing all of the required software will be downloaded, and converted to a Singularity image before execution of the pipeline.
+
+Before running the pipeline you will need to load Nextflow and Singularity using the environment module system on UzL OMICS cluster. You can do this by issuing the commands below:
+
+```bash
+module load nextflow
+module load singularity
```

Comment:
Are loaded modules carried through to jobs or should this be in a `process.beforeScript` as well?

Reply:
The modules are carried through. It's not needed in a `process.beforeScript`